src/olympia/reviewers/views.py

```diff
@@ -186,12 +186,16 @@ def dashboard(request):
     view_all = any(
         acl.action_allowed_for(request.user, perm) for perm in view_all_permissions
     )
-    queue_counts = fetch_queue_counts(request)
+    queue_counts = {
+        tab: queue.get_queryset(request).optimized_count()
```

Comment:
here you are using the key from the `reviewer_tables_registry` and in the cron you are using the name property on the value.. why not use one or the other in both places? Do we actually need the name property if we already have a key from .items()

Reply:
Yeah I don't love this, but here is the reason to my madness:
- The registry being a dict matter because we need to easily be able to find a queue by name in `queue()` (it does `TableObj = reviewer_tables_registry[tab]`) and also to show the correct tab in `queue.html`
- We need a `name` on the table anyway because we need to name the URL in `reviewers/urls.py` - that is also referenced in `queue.html`:
```
<ul class="tabnav">
  {% for this, loc, text in queue_tabnav(registry) %}
  <li class="{% if tab==this %}selected{% endif %}"><a href="{{ url('reviewers.%s' % loc) }}">{{ text }}</a></li>
  {% endfor %}
</ul>
```

Previously, there was a bit of duplication between `urlname` being set and used for URLs, but the name of the tab defined elsewhere, in the registry, and sometimes the values were close, sometimes not, which was super confusing. So this is my attempt to reconcile the 2...

You're right that I made `dashboard` inconsistent with the cron. I could use `queue.name` instead of `tab` here for consistency, that seems better.